### DISCUSSION TITLE
debian和manjaro中安装v2raya导致死机的一种情况

### DISCUSSION THREAD

**Discussion by GrandGovernor-YellowStone** [v2rayA/v2rayA#1357] [General]
v2raya作为服务在后台，使用无线连接手机热点，如果手机离开主机超过半小时以上，则debian或manjaro桌面死机（鼠标键盘无反应，cpu风扇转速飞快，无法进入tty）。查询日志反映是v2raya killed by OOM killer.

推测是v2raya无法获得网络连接后疯狂连接耗用系统资源，导致死机。请修复这个bug.

**Comment by GuoYuefei** [comment]
我这边也是，用了一小段时间后，cpu飙升，整个系统cpu资源不飙升前稳定8%，飙升后80%
arch linux
v2raya version is 2.2.5.6

**Comment by GuoYuefei** [comment]
![image](https://github.com/user-attachments/assets/83eb0aea-010e-445a-942b-e7a16e850fa7)
我来补充下截图，太夸张了；原本是一直保持运行状态的，现在是如无必要不开了

**Comment by Solitaron** [comment]
我的天，我说难怪为什么opensuse总是在无人值守的情况下死机，只要人在频繁操控系统就不会死机

**Comment by FredyVia** [comment]
请问这个问题修好了吗？ @mzz2017

**Reply by Arnold4869** [reply]
我昨天群晖又经历一次卡死，但不确定是不是这个导致的，当时系统都进不去，只看到内存占用基本满了，最后重启系统后好了

**Comment by GuoYuefei** [comment]
#1492
